language/move-lang/src/parser/syntax.rs

```diff
@@ -485,6 +485,24 @@ fn parse_value<'input>(tokens: &mut Lexer<'input>) -> Result<Value, Error> {
     Ok(spanned(tokens.file_name(), start_loc, end_loc, val))
 }
 
+// Parse a num value.
```

Comment:
To follow the existing naming convention (compare parse_address), this should be parse_num. Can you also add a grammar comment? "Num = \<NumValue\>"

Reply:
While you're at it, if you wouldn't mind, the grammar comment for parse_term should have a "\<Num\>" option listed after "\<Value\>"

Reply:
Done. Also only now push a commit (comments went accidentally before).